interface/reports/dictation_review.php

```diff
@@ -0,0 +1,316 @@
+<?php
+/**
+ * Review Dictations
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Core\Header;
```

Comment:
Although I am not sure if this is necessary, I have been placing the use commands below the require_once stuff. Just seems like that gives us more control (ie. the use does not encompass the scripts brought in via require_once which may have use commands of their own).

Reply:
k